jbmc/unit/java_bytecode/java_bytecode_parse_generics/parse_derived_generic_class.cpp

```diff
@@ -187,7 +187,9 @@ SCENARIO(
 
     const class_typet &derived_class_type =
       require_type::require_complete_java_implicitly_generic_class(
-        derived_symbol.type, {"java::ContainsInnerClassGeneric::T"});
+        derived_symbol.type,
```

Comment:
Double-check: do any of these tests exercise triple-nesting?
```
public class A<T> {
  class B {
    class C<T> {

    }
  }
}
```
That sort of thing?

Reply:
The other test in that commit does: https://github.com/diffblue/cbmc/pull/4929/commits/aec9e89c82c9042b70342de9b9bedee96c5a99b4#diff-157ff943932394172d454093024a926cR169